src/Elm/Print/Encoder.hs

```diff
@@ -18,7 +18,7 @@ import Data.Text.Prettyprint.Doc (Doc, brackets, colon, comma, concatWith, dquot
                                   equals, lbracket, line, nest, parens, pretty, rbracket, surround,
                                   vsep, (<+>))
 
-import Elm.Ast (ElmAlias (..), ElmConstructor (..), ElmDefinition (..), ElmPrim (..),
```

Comment:
We might need to add CPP similar to https://github.com/Holmusk/servant-docs-simple/blob/6f6d4e35cbce4d0dc0254a4799322ba5904f0b4b/src/Servant/Docs/Simple/Render.hs#L84-L92 to get rid of the above warnings, but this can be done in separate PR.

Reply:
I've noticed but assumed this is unrelated to the changes made, is it?

Reply:
It's deprecation in the module naming in prettyprinter library. I'll take a look at fixing these and bumping versions of GHC in CI after you merge all your PRs.

Reply:
sure no rush. I think there are bunch of things that we should batch together since renaming of constructors is breaking change we will need major version bump so I think we should not rush release and take our time improving as much as we can.